src/main/java/com/example/demo/service/SQLMethods.java

```diff
@@ -0,0 +1,89 @@
+package com.example.demo.service;
+
+import com.example.demo.entity.Team;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import java.sql.*;
```

Comment:
Avoid import from whole package.
You force developers to use API that may be not needed here.